ui/lib/core/addon/components/json-editor.hbs

```diff
@@ -10,6 +10,18 @@
         </label>
         <ToolbarActions>
           {{yield}}
+          {{#if @example}}
+            <button
+              type="button"
+              class="toolbar-link"
+              disabled={{not @value}}
```

Comment:
If there is no value the example is showing so there is nothing to restore. As soon as the user makes any change to the example the value is set and then the original example can be restored.

I think I see your point though. If the user clears out the input the example will display which may lead them to believe that it will be saved which is not the case. I think I need more clarity from design on how this should function. We could show the editor as empty if there is no value set and allow the example to be restored but I'm also wondering now if restoring the example should also set it to the value or if we wait for explicit input in the text area.

Reply:
Quick follow up I found this in the designs which seems to answer the question:

> LDIF STATEMENTS
We will provide some default example here, but those are considered as comments and we won’t submit them during create

I think we are treating the examples as a placeholder when the user has not input a value.